.github/workflows/iwyu.yml

```diff
@@ -0,0 +1,70 @@
+name: IWYU
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 12 * * 0"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  iwyu_ubuntu_2204:
+    runs-on: ubuntu-latest
+    container: ghcr.io/acts-project/ubuntu2204_clang:v41
+    steps:
+      - name: Install git lfs and llvm dev
+        run: apt-get update && apt-get install -y git-lfs llvm-dev libclang-dev clang
+
+      - name: Get IWYU source
+        uses: actions/checkout@v3
+        with:
+          repository: include-what-you-use/include-what-you-use
+          ref: clang_14
+          path: iwyu
+
+      - name: Build IWYU
```

Comment:
Would it be worth adding a docker image for this? How long does the compilation take? I suppose on schedule it doesn't really matter.

Reply:
yeah I think for now it is fine. the build is actually quite fast